package/src/languageService/kustoLanguageService.ts

```diff
@@ -674,12 +674,21 @@ class KustoLanguageService implements LanguageService {
 
         const diagnostics = blocks
             .map((block) => {
-                const diagnostics = this.toArray<Kusto.Language.Diagnostic>(block.Service.GetDiagnostics());
-                if (diagnostics) {
-                    return diagnostics;
+                let diagnostics = this.toArray<Kusto.Language.Diagnostic>(block.Service.GetDiagnostics(false));
```

Comment:
lets rename diagnostics to errorsDiagnostics 
and analyzerDiagnostics to warningAndSuggestionDiagnostics